src/ContainerTerminal.jsx

```diff
@@ -22,11 +22,30 @@ import PropTypes from 'prop-types';
 import cockpit from 'cockpit';
 import { Terminal } from "xterm";
 
-import * as utils from './util.js';
+import * as client from './client.js';
 
 import "./ContainerTerminal.css";
 
 const _ = cockpit.gettext;
+const decoder = cockpit.utf8_decoder();
+const encoder = cockpit.utf8_encoder();
+
+function sequence_find(seq, find) {
+    let f;
+    const fl = find.length;
+    let s;
+    const sl = (seq.length - fl) + 1;
+    for (s = 0; s < sl; s++) {
```

Comment:
Maybe at least the outer loop could be replaced with [findIndex()](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex). But utterly unimportant.